tests/suites/test_suite_psa_crypto.data

```diff
@@ -6984,27 +6984,27 @@ depends_on:PSA_WANT_ALG_CTR:PSA_WANT_KEY_TYPE_AES
 generate_key:PSA_KEY_TYPE_AES:64:PSA_KEY_USAGE_EXPORT | PSA_KEY_USAGE_ENCRYPT | PSA_KEY_USAGE_DECRYPT:PSA_ALG_CTR:PSA_ERROR_INVALID_ARGUMENT:0
 
 PSA generate key: RSA, 512 bits, good, sign (PKCS#1 v1.5)
-depends_on:PSA_WANT_ALG_RSA_PKCS1V15_SIGN:PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_GENERATE:MBEDTLS_RSA_MIN_KEY_SIZE <= 512
+depends_on:PSA_WANT_ALG_RSA_PKCS1V15_SIGN:PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_GENERATE:PSA_VENDOR_RSA_GENERATE_MIN_KEY_BITS <= 512
```

Comment:
Please squash b87a4aed9da9b07e4c17bf3773f2fd7aa9f6751a into bef3301ed0189299df15ec97f9b7122e3f22c9a3. As it stands now, the commit sequence doesn't match the actual history of the code, so b87a4aed9da9b07e4c17bf3773f2fd7aa9f6751a is incomprehensible to a reader who doesn't have the rebase history available. Furthermore, bef3301ed0189299df15ec97f9b7122e3f22c9a3 doesn't build in the default configuration, which makes `git bisect` painful.

Reply:
Done